packages/@lwc/engine-core/src/framework/template.ts

```diff
@@ -173,6 +174,14 @@ function serializeAttribute(part: VStaticPart, name: string) {
     return value;
 }
 
+function serializeClass(part: VStaticPart, classToken: string) {
+    const classMap = getMapFromClassName(part.data?.className);
```

Comment:
The reason for exposing the `getMapFromClassName` is to consolidate the logic for handling the `className` attribute.

We could alternatively use a simplified version of `getMapFromClassName` here.